src/util.js

```diff
@@ -0,0 +1,31 @@
+function status(checkbox, tasklist) {
+  if (checkbox.checked) {
+    tasklist.completed = true;
+  } else {
+    tasklist.completed = false;
+  }
+}
+
+function editTask(text, task, taskArray) {
+  task.description = text.innerHTML;
+  text.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      task.description = text.innerHTML;
+      localStorage.setItem('tasks', JSON.stringify(taskArray));
+      text.parentElement.classList.remove('inputEdit');
+      text.setAttribute('contenteditable', 'false');
+    }
+  });
+}
+
+function clearChecked(taskArray) {
+  taskArray = taskArray.filter((task) => !task.completed);
+  taskArray.forEach((task) => {
+    task.index = taskArray.indexOf(task) + 1;
+  });
+  localStorage.setItem('tasks', JSON.stringify(taskArray));
+}
+
+export {
+  editTask, clearChecked, status,
+};
```

Comment:
- [ ] Well done implementing these tests. I'd like to however point out that you are required to test the actual functions that you use in your Todo App. don't create new functions that you test. the purpose of the test is to test the functionality that your code is using please make the tests test the functions that your app actually uses.